### PR TITLE
Pool fixes

### PR DESCRIPTION
**Pull request by philipstanislaus**
Addresses all points in #489 except (iv) and (v). See commits for details.

### CODE REVIEW

**Comment by philipstanislaus** [comment]
Preview: https://5f5266b9a255b30e9439cd90--tinlake-staging-kovan.netlify.app/

File: components/Overview/index.tsx [hunk 1]
```diff
@@ -136,19 +136,42 @@ class Overview extends React.Component<Props> {
...
+                    <strong>Interested in investing or want to learn more?</strong>
+                    {selectedPool.website && (
+                      <>
+                        <br />
+                        <a href={selectedPool.website} target="_blank">
+                          {new URL(selectedPool.website).hostname}
+                        </a>
+                      </>
+                    )}
+                    {selectedPool.website && (
```
Comment (offerijns) [inline components/Overview/index.tsx:152]: I think this should be `{selectedPool.email && (` right?

Comment (offerijns) [inline components/Overview/index.tsx:152]: I think this should be `{selectedPool.email &&` ( right?

File: components/Overview/index.tsx [hunk 2]
```diff
@@ -136,19 +136,42 @@ class Overview extends React.Component<Props> {
...
+                      <>
+                        <br />
+                        <a href={`mailto:${selectedPool.email}`} target="_blank">
+                          {selectedPool.email}
+                        </a>
+                      </>
+                    )}
+                    {selectedPool.additionalContactInfo?.map((ci, i) =>
+                      ci.link ? (
+                        <React.Fragment key={ci.label + i}>
```
Comment (offerijns) [inline components/Overview/index.tsx:162]: I personally think the short syntax, `<>{...children}</>`, is cleaner. What do you think? 

https://reactjs.org/docs/fragments.html#short-syntax

File: config.ts [hunk 3]
```diff
@@ -178,3 +199,19 @@ const config: Config = {
 }
 
 export default config
+
+function between1e23and1e27(s: string): boolean {
```
Comment (offerijns) [inline config.ts:203]: 👍

File: utils/upcomingPoolToPoolData.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { PoolData } from '../ducks/pool'
+import BN from 'bn.js'
+import { Tranche } from '@centrifuge/tinlake-js'
+import { UpcomingPool } from '../config'
+
+const emptyTranche: Tranche = {
```
Comment (offerijns) [inline utils/upcomingPoolToPoolData.ts:6]: Good idea to separate this into a util file!

**Review by offerijns — APPROVED** [review]
A few minor comments, but looks great!

**Pull request merged**